Sources/TuistKit/GraphMappers/StaticLibWithResourcesGraphMapper.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import TuistCore
+import TSCBasic
+
+class StaticLibWithResourcesGraphMapper: GraphMapping {
+    func map(graph: Graph) throws -> (Graph, [SideEffectDescriptor]) {
+        
+        
+        
+        return (graph, sideEffects(graph))
+    }
+    
+    fileprivate func sideEffects(_ graph: Graph) -> [SideEffectDescriptor] {
+        graph.targets.flatMap({$0.value}).compactMap { (target) -> SideEffectDescriptor? in
+            switch target.target.product {
+            case .framework, .staticFramework, .staticLibrary, .dynamicLibrary:
+                let content = """
+import Foundation
+
+extension Bundle {
```

Comment:
I don't understand the template of the content, don't you want to extend `Bundle` to expose a public variable to access itself?